### ISSUE TITLE
Stage 2 reminders do not align with their assignments

### ISSUE THREAD

**Issue by nashiro** [EmotionCognitionLab/CALM#30]
I received the following reminders during Stage 2 at 10AM and 6PM. Since mindfulness practice hasn't started, it may be confusing for participants to receive these messages.

- Hello! It's time for your daily mindfulness practice. Keep calm and carry on!
- This is a friendly reminder from the CALM study team! If you haven’t done your mindfulness practice today, why not start now?
- Hi there! This is your daily reminder to do your daily mindfulness practice. Log in now for a quick 18-minute practice.

**Comment by noah10** [comment]
@cho890 - If you think this is a problem, please provide alternative text. It would be best if we could use the same text for stage 2 and stage 3. If you don't think this is a problem I'll go ahead and close this.

**Comment by cho890** [comment]
I agree, it does seem confusing.  Could we have different messages for stage 2 versus stage 3?

**Comment by noah10** [comment]
We can differentiate them later if we want, but with only one week to go before launch we should just change the text, rather than giving ourselves a new aspect of reminders to test with so little time left.

**Comment by cho890** [comment]
I would prefer not to change the text, as it is relevant for Stage 3, which is the majority of the study.

**Comment by noah10** [comment]
OK - I've marked this wontfix but I'll leave it open in case we want to revisit it later.
